File: VentasVB/cuVentas.vb
Public Class cuVentas
    Dim con As Conexion = New Conexion()
    Private Sub cuVentas_Load(sender As Object, e As EventArgs) Handles MyBase.Load
        dgvDatos.DataSource = con.datosVenta()
    End Sub

    Private Sub btnEliminar_Click(sender As Object, e As EventArgs) Handles btnEliminar.Click
        con.eliminarTabla("venta", Convert.ToInt32(dgvDatos.SelectedRows(0).Cells(0).Value), idUsuario)
        dgvDatos.DataSource = con.datosVenta()
    End Sub

End Class

File: Ventas/cuVentas.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace Ventas
{
	public partial class cuVentas : UserControl
	{
		Conexion con = new Conexion();
		public cuVentas()
		{
			InitializeComponent();
			dgvDatos.DataSource = con.datosVenta();
		}

		private void btnEliminar_Click(object sender, EventArgs e)
		{
			con.eliminarTabla("venta", Convert.ToInt32(dgvDatos.SelectedRows[0].Cells[0].Value), global.id);
			dgvDatos.DataSource = con.datosVenta();
		}
	}
}
